main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -1895,7 +1895,7 @@ public static void setCacheFilterConfig(final String config) {
     }
 
     public static int getListInitialLoadLimit() {
-        return getInt(R.string.pref_list_initial_load_limit, 0);
+        return getInt(R.string.pref_list_initial_load_limit, R.integer.list_load_limit_default);
```

Comment:
R.integer.* is just a unique ID.

It needs to be wrapped with `getKeyInt(R.integer.*)`

Reply:
Ok.
So I create another PR because this one is already merged.

Reply:
I just quickly created PR  #11289 for it myself, so that the issue doesn't get lost...